charms/istio-gateway/src/charm.py

```diff
@@ -46,7 +46,7 @@ def start(self, event):
             return
 
         if not self.model.relations["istio-pilot"]:
-            self.model.unit.status = BlockedStatus("Waiting for istio-pilot relation")
+            self.model.unit.status = BlockedStatus("Add required relation to istio-pilot")
```

Comment:
Looks weird to me as status message. It is more an instruction. May be "Waiting for missing istio-pilot relation" should be better?

Reply:
@i-chvets no
that is the point. Blocked status is a status which requires manual operation from human operator. Thus, we give an instruction what to do in order to unblock

Reply:
It shouldn't say "waiting" like it did before, but I'm kinda on the fence about these action statuses that you're suggesting @beliaev-maksim. they come across odd grammatically. maybe we can discuss in daily